plugins/cpu/cpu.go

```diff
@@ -13,31 +14,71 @@ const (
 	niceGauge
 )
 
+type average struct {
+	Sum   float64
+	Count float64
+}
+
+func (m *average) update(v float64) {
```

Comment:
I wonder if its better to use some other calculation that average? The problem was median is that it can hide spikey CPU load which indicates our systems are at capacity for a period of time. For example, https://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average